docs/incentivizedtestnet/02-phases.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Phases
+---
+
+## Phase 1 - Build-to-Earn
+Welcome to zkVerify's Incentivized Testnet - Phase 1. In this Phase, we have two exciting challenges designed to enhance blockchain privacy and scalability:
+
+- <b>Build a verification pallet:</b> dive into the core of zkVerify by creating a crucial component for our Substrate-based chain. This task involves developing a pallet that efficiently handles the verification of zero-knowledge proofs. zkVerify can accept proofs directly, they don't need to be wrapped in a bn254 compatible zk-proof. We want verifiers for all types of proofs, including STARK's and the new Elliptic Curve. 
```

Comment:
```suggestion
- <b>Build a verification pallet:</b> dive into the core of zkVerify by creating a crucial component for our Substrate-based chain. This task involves developing a pallet that efficiently handles the verification of zero-knowledge proofs. 
```
Don't like much this final piece, probably it's better not saying anything at all